task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -84,8 +84,8 @@ spec:
 
         ### Try to extract binaries with configs > check binaries functionality > check opm validate ###
         if [ ! -s ../inspect-image/image_inspect.json ]; then
-          echo "File $(workspaces.source.path)/hacbs/inspect-image/image_inspect.json did not generate correctly. Check inspect-image task log."
```

Comment:
I am confused here. Doesn't the `source` represent the workspace name of task `inspect-image` in https://github.com/redhat-appstudio/build-definitions/blob/main/task/inspect-image/0.1/inspect-image.yaml#L33, is it a right path if we change to `$(workspaces.workspace.path)` here? @jinqi7 @dirgim @MartinBasti

Reply:
I think that is the case, yes.